### ISSUE TITLE
the acc of small batch datasets is too low

### ISSUE THREAD

**Issue by binbinjiang0505** [MCG-NJU/VideoMAE#84]
As you said in your article，“ We demonstrate that VideoMAE is a data-efficient learner that could be successfully trained with only 3.5k videos.”
I use 8 GPUs, batch_size is 8 ,17400 video datasets，and the training is 800epochs during training, but the accuracy is very low, even less than 1% 。
Should I adjust my learning rate？Looking forward to your reply

**Comment by yztongzhan** [comment]
Hi @binbinjiang0505 ! Please refer to our scripts of [UCF-101](https://github.com/MCG-NJU/VideoMAE/blob/main/MODEL_ZOO.md#ucf101).

**Comment by yztongzhan** [comment]
Hi @binbinjiang0505 ! Any update?

**Comment by binbinjiang0505** [comment]
> Hi @binbinjiang0505 ! Any update?

I tested various issues, and during the training process, there was a significant decrease in train_loss. At first, I thought it was overfitting, so I used the training set to test, but the accuracy was still around 0.5%, and 1/174 ≈ 0.5%. Therefore, I currently believe that there was a problem when saving the model. The size of my saved models for every 20 epochs was 1.05g, resulting in each saved model being maybe a model that I hadn't learned initially., But I haven't modified any code, so I'm very confused now. (The dataset I used is SSV2, with 100 videos selected for each category constituting 17400 video datasets)
